scripts/transition_metrics.py

```diff
@@ -279,6 +284,39 @@ def commodity_to_prototype(transactions_df, commodity, prototype):
     return prototype_transactions
 
 
+def commodity_to_LWR(transactions_df, commodity, prototype):
+    '''
+    Finds the transactions of a specific commodity sent to the LWRs in the simulation,
```

Comment:
Sorry to be so nitpicky, this is the last thing. This line is too long:
```suggestion
    Finds the transactions of a specific commodity sent to the LWRs in the 
```